8Knot/queries/prs_query.py

```diff
@@ -28,7 +28,11 @@ def prs_query(self, repos):
     if len(repos) == 0:
         return None
 
-    query_string = f"""
+    # NOTE: This was dropping PRs that were needed in the heatmap
+    # visualizations, so we switched back to the old query.
```

Comment:
wdym switched back to the old query? What was the new one exactly?

Reply:
bad comment, will remove